README.md

```diff
@@ -3,23 +3,19 @@
 <img align="left" src="https://avatars.githubusercontent.com/u/125200832?s=128&v=4" alt="ratatui logo">
 
 This is the documentation project for [Ratatui](https://github.com/tui-rs-revival/ratatui), a Rust
-library to build rich terminal user interfaces and dashboards. The documentation can be read at
-<https://tui-rs-revival.github.io/ratatui-book/>.
+library to build rich terminal user interfaces and dashboards. Note that this documentation is supplementary
+to the technical documentation, and we also maintain crate documentation. This is more of a guide book
+to help our users understand how to build a TUI with Ratatui. If you are looking for documentation on
+structs, enums, traits, methods, etc., please visit our [crates.io documentation](https://docs.rs/ratatui/0.21.0/ratatui/).
```

Comment:
> structs, enums, traits, methods, etc.

I would skip this listing in favour of something like "technical documentation" or "crate documentation" as you used earlier.

> please visit our  [crates.io documentation](https://docs.rs/ratatui/0.21.0/ratatui/)

Crate documentation ? or maybe docs.rs documentation ? crates.io documentation sounds a bit wrong to me, what do you think ? Of course I'm nitpicking here.